.babelrc

```diff
@@ -11,7 +11,7 @@
     "annotate-pure-calls",
     "@babel/plugin-syntax-object-rest-spread",
     "transform-react-remove-prop-types",
-    "@babel/plugin-syntax-dynamic-import",
+    "dynamic-import-node",
     "@babel/plugin-transform-runtime",
```

Comment:
whY?

Reply:
because tests was failing for dynamic imports(youtube element)